app/containers/ITunesDeatails/tests/index.test.js

```diff
@@ -0,0 +1,24 @@
+/**
+ *
+ * Tests for ITunesDeatails container
+ *
+ *
+ */
+
+import React from 'react';
+// import { fireEvent } from '@testing-library/dom';
+import { renderProvider } from '@utils/testUtils';
+import { ITunesDeatailsTest as ITunesDeatails } from '../index';
```

Comment:
Please fix the typo in details

Reply:
I have fixed this in the tests/song-details branch. Will be fixed when code gets pushed from that branch